packages/webpack/src/config/base.js

```diff
@@ -225,8 +225,10 @@ export default class WebpackBaseConfig {
       {
         test: /\.jsx?$/i,
         exclude: (file) => {
+          file = file.split('node_modules', 2)[1]
```

Comment:
What if `node_modules` is not included in that path? 🤔

(Is that even possible?)

Reply:
If it doesn't include `node_modules`, then it won't be included in loader, same as before